### DISCUSSION TITLE
Emoji Search

### DISCUSSION THREAD

**Discussion by B1naryB0t** [florisboard/florisboard#2399] [General]
How feasible is it to implement an emoji search feature? It would really help usability

**Comment by ghost** [comment]
id love to see this to even tho this post is a year old an it prob would of been added by now

**Reply by chailxx** [reply]
Was literally coming here to suggest this🙏😭

**Comment by klaurence** [comment]
There technically is an emoji search functionality since v0.4.1. You can use it by typing a colon (:), then at least three letters of the emoji's name. My choice settings are "type at least one letter" and "show the name of the emoji being shown", which you can set in `Settings > Emojis`.
